doc/releases/changelog-dev.md

```diff
@@ -414,49 +437,32 @@ of operators. [(#2622)](https://github.com/PennyLaneAI/pennylane/pull/2622)
 * Add trivial behaviour logic to `qml.operation.expand_matrix`.
   [(#2785)](https://github.com/PennyLaneAI/pennylane/issues/2785)
 
-* Adds a new function to compare operators. `qml.equal` can be used to compare equality of parametric operators taking
-  into account their interfaces and trainability.
+* Adds a new function to compare operators. `qml.equal` can be used to compare equality 
+  of parametric operators taking into account their interfaces and trainability.
   [(#2651)](https://github.com/PennyLaneAI/pennylane/pull/2651)
 
 * The `default.mixed` device now supports backpropagation with the `"jax"` interface.
```

Comment:
New feature?

Reply:
Yes we should move it to new feature.

Reply:
Would need a code example I think for this as well